deploy-config/staging.yml

```
@@ -1,8 +1,10 @@
 env: staging
 instances: 1
 memory: 1G
+command: gunicorn -c /home/vcap/app/gunicorn_config.py application
```

And staging should have it as well:

```suggestion
command: newrelic-admin run-program gunicorn -c /home/vcap/app/gunicorn_config.py application
```